### PR TITLE
Updated actions to be docker based and restructured file tree

### PR DESCRIPTION
**Pull request by ghost**
All commits tested on local fork, this is a big delta because we are now moving dev to the staging repos

### CODE REVIEW

File: docs/1_Landing.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+The tools in this SDK make the LUCI platform, sensor information and a control interfaces, available to organizations and leading research institutions, allowing them to use LUCI as a research tool. To use the SDK you will need a LUCI enabled chair updated to LuciCore Sandbox Edition, Version 1.6.2 or greater. If you are interested in gaining access to the Sandbox, contact info@luci.com.
+
+PLEASE NOTE THE SDK IS CURRENTLY CHANGING RAPIDLY DURING DEVELOPMENT
+
+
+# Overview
+
+LUCI provides an SDK for use of LUCI within the ROS2 ecosystem and compatibility with the open Wheelchair Digital Interface (WDI) standard to enable research. 
+
+For ROS2, LUCI provides multiple deb packages containing message types, executable nodes, and URDF who’s entire purpose is to get LUCI data into ROS2 compatible formats. As LUCI uses a different internal messaging system than ROS, the bulk of the ROS2 SDK is translating between message types.
```
Comment (lupak) [inline docs/1_Landing.md]: who's -> whose

**Review by lupak — DISMISSED** [review]

File: deployment-scripts/deploy.sh [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+# git clone -b ${DIGITAL_INT_BRANCH} https://${ACCESS_TOKEN}@github.com/lucimobility/wheelchair-digital-interface $DIGITAL_INTERFACE_REPO
+
+# Start with fresh file structure
+rm -rf $DOCS_REPO/source_files
+mkdir $DOCS_REPO/source_files
+mkdir $DOCS_REPO/source_files/'1_ROS2 SDK'
+mkdir $DOCS_REPO/source_files/'2_Using Encoders'
+
+# Uncomment when digial interface is ready to be added
+mkdir $DOCS_REPO/source_files/'3_Wheelchair Digital Interface'
```
Comment (ghost) [inline deployment-scripts/deploy.sh]: should this be commented?

File: deployment-scripts/deploy.sh [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+cp -r ../docs/*.md $DOCS_REPO/source_files/
+cp -r $ENCODER_REPO/docs/* $DOCS_REPO/source_files/'2_Using Encoders'
+
+# Uncomment when digial interface is ready to be added
+# cp -r ${DIGITAL_INTERFACE_REPO}/docs/* $DOCS_REPO/source_files/'3_Wheelchair Digital Interface'
+# cp -r ${DIGITAL_INTERFACE_REPO}/README.md $DOCS_REPO/source_files/'3_Wheelchair Digital Interface'
+
+
+# Generate the static site
+
```
Comment (ghost) [inline deployment-scripts/deploy.sh]: can some of these extra lines be removed

File: deployment-scripts/deploy.sh (hunk 2 above)
Comment (ghost) [reply deployment-scripts/deploy.sh]: Fixed

File: deployment-scripts/deploy.sh (hunk 3 above)
Comment (ghost) [reply deployment-scripts/deploy.sh]: Removed

**Review by lupak — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**